modules/og_context/config/schema/og_context_config.schema.yml

```diff
@@ -0,0 +1,12 @@
+og_context.og_context_config.*:
+  type: config_entity
+  label: 'OG context config config'
```

Comment:
This has the word "config" twice in the label :) Also not needed to prefix the config ID with `og_context_`.